autoarray/mock/fixtures.py

```diff
@@ -356,6 +359,28 @@ def make_regularization_constant():
     return Constant(coefficient=1.0)
 
 
+def make_regularization_constant_split():
+    return ConstantSplit(coefficient=1.0)
+
+
+def make_regularization_adaptive_brightness():
+    return AdaptiveBrightness(
+        inner_coefficient=0.1,
+        outer_coefficient=10.0,
+        signal_scale=0.5,
+        if_interpolated=False,

```

Comment:
In the `autoarray.mock` module we have code which creates standarized mock objects, so we can construct things like an `Inversion` is just a few lines of code.

Here, I add some mock fixtures for the new regularization schemes, which I will use to test `Inversions`.